flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannelTest.java

```diff
@@ -170,6 +170,65 @@ public Void call() throws Exception {
 		}
 	}
 
+	@Test
+	public void testConcurrentNotifyBufferAvailableAndRelease() throws Exception {
```

Comment:
Hmmm, isn't this cover by for example `org.apache.flink.streaming.runtime.io.benchmark.StreamNetworkThroughputBenchmarkTest#largeRemoteMode`? Maybe we do not need this test after all?

Reply:
sorry, I did not miss it but forgot to react...

There are two things that I don't like about removing this test:
- how is `StreamNetworkThroughputBenchmarkTest#largeRemoteMode` a stress test for concurrently releasing and notifying about new buffers?
- a benchmark should be about performance while a test should cover correctness: even if a performance test does cover correctness at the moment, that could too easily be changed in future